### PR TITLE
fix benchmark failure with numpy 1.20+

### PR DESCRIPTION
**Pull request by simonjayhawkins**
xref https://github.com/pandas-dev/pandas/pull/38379#pullrequestreview-580260176

This will be blocking using numpy 1.20 on Github actions Checks, xref https://github.com/pandas-dev/pandas/pull/36092#issuecomment-771112784

### CODE REVIEW

File: asv_bench/benchmarks/series_methods.py [hunk 1]
```diff
@@ -135,7 +135,7 @@ def time_isin_long_series_long_values_floats(self):
 
 class IsInLongSeriesLookUpDominates:
     params = [
-        ["int64", "int32", "float64", "float32", "object", "Int64", "Float64"],
+        ["int64", "int32", "float64", "float32", "object"],
```
Comment (jorisvandenbossche) [inline asv_bench/benchmarks/series_methods.py:138]: Can you instead raise a NotImplementedError in the `setup` below if `dtype in ("Int64", "Float64") and np.__version >+ "1.20.0"` ?

That keeps the benchmark running for if you don't have numpy 1.20 installed (although I don't know how often the environment gets updated on our benchmark server)

Comment (simonjayhawkins) [reply asv_bench/benchmarks/series_methods.py]: Thanks @jorisvandenbossche . done.

Comment (jorisvandenbossche) [reply asv_bench/benchmarks/series_methods.py]: Thanks!

Further follow-up question: 1) is there actually an issue about this failure? 2) can you add a link to that issue as a `TODO` comment where raising the NotImplementedError?

**Comment by jreback** [comment]
thanks @simonjayhawkins

**Comment by jreback** [comment]
@meeseeksdev backport 1.2.x

File: asv_bench/benchmarks/series_methods.py (hunk 1 above)
Comment (simonjayhawkins) [reply asv_bench/benchmarks/series_methods.py]: > is there actually an issue about this failure?

#39844



> can you add a link to that issue as a `TODO` comment where raising the NotImplementedError?

as a follow-up

**Comment by simonjayhawkins** [comment]
changed milestone, backport not needed. see https://github.com/pandas-dev/pandas/pull/39842#issuecomment-779915656

**Pull request merged**